source/com.google.firebase/firebase-firestore/Transforms/Metadata.xml

```diff
@@ -339,4 +320,40 @@
         Firebase.Firestore.Proto
     </attr>
 
+
+    <attr
+        path="/api/package[@name='com.google.firestore.bundle']"
+        name="managedName"
+        >
+        Firestore.Bundle

```

Comment:
We also have:
```
    <attr
        path="/api/package[@name='com.google.firestore.proto']"
        name="managedName"
        >
        Firebase.Firestore.Proto
    </attr>
```

Should we make this `Firebase.Firestore.Bundle`?

Reply:
yes!

Reply:
> yes!

Ups. Not so sure. I scratched my head for a while too.
Those are 2 completely different namespaces and I tried to obey convention for renaming:

https://github.com/xamarin/GooglePlayServicesComponents/blob/create-pull-request/patch-1619417234/source/com.google.firebase/firebase-firestore/Transforms/Metadata.xml#L310-L329

```
    <attr
        path="/api/package[@name='com.google.firebase.firestore.bundle']"
        name="managedName"
        >
        Firebase.Firestore.Bundle
    </attr>
    <attr
        path="/api/package[@name='com.google.firestore.proto']"
        name="managedName"
        >
        Firebase.Firestore.Proto
    </attr>


    <attr
        path="/api/package[@name='com.google.firestore.bundle']"
        name="managedName"
        >
        Firestore.Bundle
    </attr>
```

if google comes up with such namespace, why changing it?

Besides it would merge 2 packagenames into 1 namespace and maybe cause conflicts...

Reply:
Yeah, that makes sense.  Merging 2 namespaces would probably be bad.  👍